### ISSUE TITLE
Fix modchat feature

### ISSUE THREAD

**Issue by palt0** [Parnassius/cerbottana#73]
Upon joining a room, cerbottana doesn't detect if `modchat +` is already set unless the action is logged in the chat.

I'm not delving into the details because I already discussed about this issue with @Parnassius and he's working on a fix.

**Comment by Parnassius** [comment]
Fixed in https://github.com/Parnassius/cerbottana/commit/71975d31c84dc89b28fff8d1de3db7ce684d7b37

**Comment by palt0** [comment]
Modchat is still bugged. This is a high priority issue.

Check today's [italiano](https://psim.us/italiano) modlog: the bot tries to set it again if a staffer removes it.

I experienced the same issue some time ago in my local testserver and it also was after 71975d31c84dc89b28fff8d1de3db7ce684d7b37. I attached the relevant logs:
[cerbottana.txt](https://github.com/Parnassius/cerbottana/files/5484706/cerbottana.txt)
![ps_screenshot](https://user-images.githubusercontent.com/67389798/98052711-2a800680-1e37-11eb-83e3-3e44269504c6.png)
The `.guessthecard` command was supposed to just `pass` without doing anything.

**Comment by palt0** [comment]
Modchat is still bugged even after closing Parnassius#73. This is a high priority issue.

Check today's [italiano](https://psim.us/italiano) modlog: the bot tries to set it again if a staffer removes it.

I experienced the same issue some time ago in my local testserver and it also was after 71975d31c84dc89b28fff8d1de3db7ce684d7b37. I attached the relevant logs:
[cerbottana.txt](https://github.com/Parnassius/cerbottana/files/5484706/cerbottana.txt)
![ps_screenshot](https://user-images.githubusercontent.com/67389798/98052711-2a800680-1e37-11eb-83e3-3e44269504c6.png)
The `.guessthecard` command was supposed to just `pass` without doing anything.
